include/Clipboard.h

```diff
@@ -29,25 +29,40 @@
 #include <QDomElement>
 
 
-class JournallingObject;
-
 class Clipboard
 {
 public:
-	typedef QMap<QString, QDomElement> Map;
-
-	static void copy( JournallingObject * _object );
-	static const QDomElement * getContent( const QString & _node_name );
-
-	static const char * mimeType()
+	enum MimeType
 	{
-		return( "application/x-lmms-clipboard" );
-	}
+		StringPair,
+		Default
+	};
 
+	// Convenience Methods
+	static const QMimeData * getMimeData();
+	static bool hasFormat( MimeType mT );
 
-private:
-	static Map content;
+	// Helper methods for String data
+	static void copyString( const QString & str, MimeType mT );
 
+	// Helper methods for String Pair data
+	static void copyStringPair( const QString & key, const QString & value );
+	static QString decodeKey( const QMimeData * mimeData );
+	static QString decodeValue( const QMimeData * mimeData );
+
+	static const char * mimeType( MimeType type )
+	{
+		switch( type )
+		{
+			case Clipboard::StringPair:
+				return( "application/x-lmms-stringpair" );
```

Comment:
```suggestion
				return "application/x-lmms-stringpair";
```
Return Without parentheses.